tests/models/yolov3/test_yolov3.py

```diff
@@ -0,0 +1,42 @@
+# Reference: https://github.com/tenstorrent/tt-buda-demos/blob/main/model_demos/cv_demos/yolo_v3/pytorch_yolov3_holli.py
+
+import torch
+from PIL import Image
+from torchvision import transforms
+import requests
+import pytest
+
+from tests.models.yolov3.holli_src.yolov3 import *
+
+
+@pytest.mark.compilation_xfail
+def test_yolov3(record_property):
+    record_property("model_name", "YOLOv3")
+
+    # Download model weights
+    url = "https://www.ollihuotari.com/data/yolov3_pytorch/yolov3_coco_01.h5"
```

Comment:
this tests downloaded neat the test .py file?

Reply:
You are right. That's not good. So I modified to download the model weights into the system `/tmp` directory.

Besides, I added to check the existence of the downloaded model weights. If the weight has been downloaded before and has not been cleared by the system, it can be reused by the tests, which saves time and bandwidth to download again, while weight is usually large.

Reply:
We carefully clean cache between model test runs to make sure we fit into the allocated disk space. 
Lets please put it inside `~/.cache/custom_weights`
I will make sure to clean this in the workflow.

Reply:
It makes sense. Changed as you described.